src/cmd-run

```diff
@@ -141,6 +141,11 @@ if [ -z "${IGNITION_CONFIG_FILE:-}" ]; then
                         "contents": "[Service]\\nTTYVTDisallocate=no\\nExecStart=\\nExecStart=-/usr/sbin/agetty --autologin core --noclear %I \$TERM\\n"
                     }
                 ]
+            },
+            {
+                "name": "serial-getty-motd.service",
+                "enabled": true,
+                "contents": "[Unit]\\nDescription=Helper message to exit QEMU monitor to /etc/motd\\nBefore=getty.target\\n[Service]\\nType=oneshot\\nExecStart=/bin/sh -c '/usr/bin/echo \\"To exit, press Ctrl-A and then X.\\" >> /etc/motd'\\n[Install]\\nWantedBy=getty.target\\n"
```

Comment:
I find adding a space between that and the prompt helps make it easier to read:

```suggestion
                "contents": "[Unit]\\nDescription=Helper message to exit QEMU monitor to /etc/motd\\nBefore=getty.target\\n[Service]\\nType=oneshot\\nExecStart=/bin/sh -c '/usr/bin/echo -e \\"To exit, press Ctrl-A and then X.\\\\n\\" >> /etc/motd'\\n[Install]\\nWantedBy=getty.target\\n"
```

Reply:
Agreed - this looks nicer, thanks! Adding in.